web-marketplace/src/pages/Projects/AllProjects/AllProjects.tsx

```diff
@@ -227,6 +233,9 @@ export const AllProjects: React.FC<React.PropsWithChildren<unknown>> = () => {
       )}
       {projects?.map(project => {
         const isSoldOut = getIsSoldOut({ project, soldOutProjectsIds });
+        const isComplianceProject = project.marketType === COMPLIANCE_MARKET;
```

Comment:
we had to change `marketType` to be a list, because some projects can are both voluntary and compliance https://github.com/regen-network/regen-data-standards/pull/25

```suggestion
        const isComplianceProject = project.marketType.includes(COMPLIANCE_MARKET);
```